chains/miraclechain/metadata.yaml

```diff
@@ -7,14 +7,16 @@ blockExplorers:
 blocks:
   confirmations: 1
   estimateBlockTime: 3
-  reorgPeriod: 5
+  reorgPeriod: 0
```

Comment:
Just to double confirm that Miracle chain produces blocks on demand, that's why we use `0` as reorg period?

Reply:
yup that's right